docs/en/data_types/datetime64.md

```diff
@@ -0,0 +1,89 @@
+# DateTime64 {#data_type-datetime64}
+
+Stores a moment in time, including date, time and timezone, with defined sub-second precision
```

Comment:
Timezone is not stored in value, only in metadata. You cannot have values with different timezones in a single column.